### DISCUSSION TITLE
How can I enable member subscription page

### DISCUSSION THREAD

**Discussion by profdorly** [Inoryum-Ltd/aesto#12] [Q&A]
Hello, folks! How are you?

I'm using the latest version of Ghost (4.7.0) and I didn't find in the template docs how can I enable the member subscription page. In the documentation, the instructions are for Ghost 3.x.x. Could you help me with that?

Thank you!

**Comment by saleheenkhan** [comment]
Hi,
Aesto now supports ghost 4.x.x. We are moving all our themes to new branding called [Visioun](https://visioun.com). thats the docs not updated yet.

To enable the members pages like Sign-in, Sign-up, accounts or the pricing page you need to upload the `routes.yaml` file . from Ghost Admin > Labs.

Once you upload the file you will see the membership related pages all are working.

I hope this helps .

Let me know if you have any more questions.

Regards,
Salehin from Inoryum Ltd

**Reply by profdorly** [reply]
Thank you for the reply! I did it (enabling the routes.yaml), configured the Stripe but the subscription page didn’t appear.

My website right now: https://www.powernap.School

What can I do?

**Reply by saleheenkhan** [reply]
Hi,
The site seems not reachable.. By subscription page you are referring to the pricing page right? if so currently the pricing page is for logged users ..

If you log-in as user & go to account page there you will see a button to pricing page. 


So if you need the pricing page to be public that needs some customizations.. I can give you instructions for that customization but if you dont want your hands dirty with the codes then we also have services for that.. Please send a DM twitter [@inoryumLTD](https://twitter.com/inoryumLTD) & We will try to help you.

**Reply by saleheenkhan** [reply]
btw i was able to visit your site without the "www"

**Reply by profdorly** [reply]
Ok! I can do the customization. Do you have these instructions to send for me? Thank you!

**Reply by profdorly** [reply]
Oh, I'm translating the template to brazilian Portuguese. When I finish it I'll send for you guys.

**Reply by saleheenkhan** [reply]
I need to create the instructions .. I will send it to you tomorrow .. sounds good ?

& Thanks for the translation
